src/routes/land/[slug]/+page.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import Accordion, { AccordionItem } from "$lib/components/Accordion";
+  import Button from "$lib/components/Button/Button.svelte";
+  import Card from "$lib/components/Card/Card.svelte";
+  import ContactCard from "$lib/components/ContactCard";
+  import ContentfulRichText from "$lib/components/ContentfulRichText";
+
+  import type { ServiceEntry, ServiceGroup } from "$lib/services/contentful/schema";
+
+  export let data;
+
+  // TODO: remove and validate core content pages with data once navigation changes are merged
+  // import pestControlData from "./pest-control-data.json";
+  // let data = pestControlData.data.serviceGroupCollection.items[0];
+
+  // import industrialHempData from "./industrial-hemp-data.json";
+  // let data = industrialHempData.data.serviceGroupCollection.items[0];
+  $: ({
+    title,
+    subheading,
+    description,
+    heroImage,
+    serviceListName,
+    serviceEntriesCollection,
+    contactInfoCollection,
+    additionalResources,
+  } = data);
+
+  let serviceEntries: ServiceEntry[] = [];
+  let serviceGroups: ServiceGroup[] = [];
+  $: serviceEntries = [];
+  $: serviceGroups = [];
+
+  $: {
+    data.serviceEntriesCollection?.items.map((item) => {
+      if (item && "entryTitle" in item) {
+        serviceEntries?.push(item);
+      } else if (item && "title" in item) {
+        serviceGroups?.push(item);
+      }
```

Comment:
A better way to determine whether the reference is a "service group" or "service entry" is to use the built-in `__typename` field; I did something similar here (because the reference could have been a "menu", "menu item", or "top tier"): 
https://github.com/la-ldaf/ldaf-site/blob/84a95a69ed8900f5e6fac5f3c74b0b5978b4cf1e/src/lib/components/Header/Nav/Nav.server.ts#L20
https://github.com/la-ldaf/ldaf-site/blob/84a95a69ed8900f5e6fac5f3c74b0b5978b4cf1e/src/lib/components/Header/Nav/Nav.server.ts#L65

If you do this in the `+page.server.ts` file you might not need the reactive statement here.

Reply:
nice, I'm much more into that. I tried messing around with `__typename` but couldn't figure it out. this works, though, so I'm pushing an update in a similar vein